Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -28,3 +28,4 @@
       - id: BoxSurvivalMedical
       - id: CheapRollerBedSpawnFolded
       - id: BodyBag_Folded
+      - id: Portafib
```

Comment:
add comment `// DeltaV - Add Portafibs`
or `// DeltaV - Add Portafibs, see Prototypes/DeltaV/Entities/Objects/Devices/Medical/portafib.yml`

Reply:
There's no need to add comments here indicating a Delta V change, when the entire file is a Delta V change 😉

Reply:
_Looks at date_
Hm yeah probably shouldn't review right after I wake up